app/tests/test_quizzes.py

```diff
@@ -0,0 +1,38 @@
+from fastapi.testclient import TestClient
+import unittest
+import json
+from mongoengine import connect, disconnect
+from main import app
+
+client = TestClient(app)
+
+
+class QuestionsTestCase(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        connect("mongoenginetest", host="mongomock://127.0.0.1:8000")
+
+    @classmethod
+    def tearDownClass(cls):
+        disconnect()
+
+    def setUp(self):
+        data = open("app/dummy_data/homework_quiz.json")
+        self.quiz_data = json.load(data)
+        response = client.post("/quiz/", json=self.quiz_data)
+        response = json.loads(response.content)
+        self.id = response["_id"]
+
+    def test_to_create_quiz(self):
+        response = client.post("/quiz/", json=self.quiz_data)
+        assert response.status_code == 201
```

Comment:
The way to test that this worked would be to make a GET request to the DB and see if the document with the same ID exists